CONTRIBUTING.md

```diff
@@ -62,8 +62,10 @@ Follow the steps below to setup the environment:
 
 - [Clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
 - Install Go by following [installation](https://go.dev/doc/install). Please check minimal go version in [go.mod file](./go.mod#L3).
-- Install build tools (`make`):
-  - For debian based distributions you can run `sudo apt-get install build-essential`
+- Install build tools:
+  - `make`: For debian based distributions you can run `sudo apt-get install build-essential`
+  - `protoc`: You can download for your os. Use version `v3.20.3`.
```

Comment:
Let's include a link to the [github release](https://github.com/protocolbuffers/protobuf/releases/tag/v3.20.3) to save folks searching for it.